eng/scripts/emitter-package.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
shouldn't need this second emitter-package.json

Reply:
yep, just deleted that